src/client/menu/videomenu.c

```diff
@@ -499,33 +508,58 @@ VID_MenuInit(void)
 
 	if (strcmp(vid_renderer->string, "gl3") == 0 || strcmp(vid_renderer->string, "gles3") == 0)
 	{
-		s_intensity_slider.generic.type = MTYPE_SLIDER;
-		s_intensity_slider.generic.name = "color intensity";
-		s_intensity_slider.generic.x = 0;
-		s_intensity_slider.generic.y = (y += 10);
-		s_intensity_slider.cvar = "gl3_intensity";
-		s_intensity_slider.minvalue = 0.1f;
-		s_intensity_slider.maxvalue = 256.0f;
-
-		s_overbrightbits_slider.generic.type = MTYPE_SLIDER;
-		s_overbrightbits_slider.generic.name = "overbrights";
-		s_overbrightbits_slider.generic.x = 0;
-		s_overbrightbits_slider.generic.y = (y += 10);
-		s_overbrightbits_slider.cvar = "gl3_overbrightbits";
-		s_overbrightbits_slider.minvalue = 0.1f;
-		s_overbrightbits_slider.maxvalue = 256.0f;
+		gl1_colorlight = 0;
+		s_gl3_intensity_slider.generic.type = MTYPE_SLIDER;
+		s_gl3_intensity_slider.generic.name = "color intensity";
+		s_gl3_intensity_slider.generic.x = 0;
+		s_gl3_intensity_slider.generic.y = (y += 10);
+		s_gl3_intensity_slider.cvar = "gl3_intensity";
+		s_gl3_intensity_slider.minvalue = 0.1f;
+		s_gl3_intensity_slider.maxvalue = 256.0f;
+
+		s_gl3_overbrightbits_slider.generic.type = MTYPE_SLIDER;
+		s_gl3_overbrightbits_slider.generic.name = "overbrights";
+		s_gl3_overbrightbits_slider.generic.x = 0;
+		s_gl3_overbrightbits_slider.generic.y = (y += 10);
+		s_gl3_overbrightbits_slider.cvar = "gl3_overbrightbits";
+		s_gl3_overbrightbits_slider.minvalue = 0.1f;
+		s_gl3_overbrightbits_slider.maxvalue = 256.0f;
```

Comment:
For `gl3_intensity` and `gl3_overbrightbits` a maximum value of 256 means that all sane values are within the first pixels of the slider. What about limiting the menu to a maxumum value of 5? If anyone really wants more they can set it through the console.